Release notes 6.6.2

amarantha-k opened the pull request.

Commit:
WIP - Release notes draft

Commit:
Release notes draft

Commit:
Add improvements and platform support

daverigby commented on modules/release-notes/pages/relnotes.adoc:
```diff
@@ -5,6 +5,286 @@
+
+[#table_fixedissues_v662-data,cols="25,66"]
+|===
+| Issue | Description
```
MB-44832 missing. Description requested from @BenHuddleston.

daverigby replied:
MB-44079 missing. Description requested from @BenHuddleston .

daverigby replied:
MB-43744 missing. Description requested from @BenHuddleston

daverigby replied:
MB-43717 missing. Description requested from @jameseh96

daverigby replied:
MB-43205 is missing. There's already a description on the MB - please can you add it here?

daverigby replied:
MB-41942 is missing. See discussion on MB.

daverigby replied:
MB-41407 is missing. Already a description on the MB - please can you add here.

daverigby replied:
MB-41406 missing. Description requested from @jameseh96

daverigby commented on modules/release-notes/pages/relnotes.adoc:
```diff
@@ -5,6 +5,286 @@
+
+[#table_fixedissues_v662-data,cols="25,66"]
+|===
+| Issue | Description
```
MB-43847 missing. Descrioption requested from @paolococchi

amarantha-k replied:
Waiting for description on the ticket.

owend74 replied:
Paolo is out today, so I have provided a summary.

Summary: Fixes an issue with ephemeral buckets where duplicate items could be sent in a backfill.

amarantha-k replied:
Thank you, @owendCB!

b33f commented on modules/release-notes/pages/relnotes.adoc:
```diff
@@ -5,6 +5,286 @@
+[#supported-platforms-662]
+=== New Supported Platforms
+
+This release adds support for the Ubuntu 20.04 platform.
```
Windows 2016 Datacenter edition also added DOC-8233

JFlath commented on modules/release-notes/pages/relnotes.adoc:
```diff
@@ -5,6 +5,286 @@
+| *Summary*: Fixed an issue where setupSSL() failed and impacted query execution.
+
+| https://issues.couchbase.com/browse/MB-43301[MB-43301^]
+| *Summary*: Fixed an issue where bulk DELETE with USE KEYS did not delete all keys.
```
@kaminijagtiani Does this text represent the up to date summary, or should this be updated?

kaminijagtiani replied:
Should be updated.
Fixed an issue wherein making a copy of a document internally wasn't being done correctly. 
In case of update, delete, and merge using update or delete it could have resulted in a data loss by deleting a wrong document. In case of join, let, unnest it could have given wrong results- if we were joining on meta().id or projecting meta().id.

amarantha-k replied:
Thanks Kamini.

owend74 commented on modules/release-notes/pages/relnotes.adoc:
```diff
@@ -5,6 +5,286 @@
+[#table_fixedissues_v662-tools-ui-rest-api,cols="25,66"]
+|===
+| Issue | Description
+
```
For Tools, please can we add

MB-44927
Summary: Updated to Python Requests 2.23.0

MB-44925
Summary: Fixed a case where cbbackupmgr would use the incorrect network port when alternative addressing was set.

owend74 commented on modules/release-notes/pages/relnotes.adoc:
```diff
@@ -5,6 +5,286 @@
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-44451[MB-44451^]
+| *Summary*: The Admin console now displays a tooltip to help distinguish very long bucket names.
```
Also could we add...

MB-44580
Summary: Fixes an issue to ensure the flag -x uncompress=1 operates as expected when using cbtransfer to transfer data out of a cluster.

Commit:
DOC-8233 Add Windows Server 2016 Datacenter Edition

Commit:
Incorporate feedback

jon-strabala committed:
Review of Eventing Stuff

MB-43762[MB-43762^] Describe the Feed Boundary or  "dcp_stream_boundary" improvement correctly, add a major improvement.
MB-43364[MB-43364^] Only occurred when undeployed from the paused state
MB-42804[MB-42804^] Only occurs via the REST API
MB-38403[MB-38403^] This is a protective improvement needed when users didn't properly code their function.
MB-44016[MB-44016^] Uncommented and described

Commit:
Update desc for MB-43301 as noted in feedback

Commit:
Update desc for MB-43847

Commit:
remove private tickets

arod1987 reviewed (approved):
Looks good for Tools, UI and XDCR

amarantha-k commented:
Reviewed and approved
